json-schema-bundler-cli.mjs

```diff
@@ -73,8 +77,15 @@ try {
     process.exit(1);
 }
 
+let result;
 if (argv.y) {
-    console.log(YAML.dump(schema, argv.p ? undefined : { flowLevel: 3 }));
+    result = YAML.dump(schema, argv.p ? undefined : { flowLevel: 3 });
 } else {
-    console.log(JSON.stringify(schema, undefined, argv.p ? 2 : undefined));
+    result = JSON.stringify(schema, undefined, argv.p ? 2 : undefined);
+}
+
+if (argv.o === '') {
+    console.log(result);
+} else {
+    fs.writeFileSync(argv.o, result);
```

Comment:
How about some error handling to catch file system related issues, read only file system for example or out of memory, to provide the user with a useful error message instead of the node error and stack trace while still providing the error exit code.